packages/labs/src/components/omnibox/omnibox.tsx

```diff
@@ -31,14 +31,19 @@ import * as Classes from "../../common/classes";
 
 export interface IOmniboxProps<T> extends IListItemsProps<T> {
     /**
+     * React child to render when query is empty.
+     */
+    initialRenderer?: JSX.Element;
```

Comment:
❌ this is not a _renderer_, it's an already rendered element. either name or type must change.

note how it's `noResults: JSX.Element` and `itemRenderer: (props) => JSX.Element`